Nef_3/include/CGAL/Nef_3/bounded_side_3.h

```diff
@@ -68,24 +99,31 @@ Bounded_side bounded_side_3(IteratorForward first,
        we don't need to care about the plane orientation */
   }
 
+  typename R::Non_zero_dimension_3 non_zero_dimension_3;
+  int dir = non_zero_dimension_3(plane.orthogonal_vector());
+#if 0
+  auto apv = approx(plane.orthogonal_vector());
 
+  int dir = 0;
+  double max = CGAL::abs(apv.x().sup());
+  if(CGAL::abs(apv.y().sup()) > max){
+    dir = 1;
+    max = CGAL::abs(apv.y().sup());
+  }
+  if(CGAL::abs(apv.z().sup()) > max){
+    dir = 2;
+  }
+#endif
 
   CGAL_assertion(!plane.is_degenerate());
-  Point_2 (*t)(Point_3);
-  Vector_3 pv(plane.orthogonal_vector()), pxy(0,0,1), pyz(1,0,0), pzx(0,1,0);
```

Comment:
Removing this line makes the previous typedef useless, leading to these warnings : https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-90.shtml#Convex_decomposition_3_Examples